src/routes/users.js

```diff
@@ -8,8 +8,8 @@ const controller = new Controller(UserTable, userValidator);
 const reqHandler = require("../middleware/req-handel");
 
 router.post("/", reqHandler(controller.create.bind(controller)));
-router.put("/:id", auth, reqHandler(controller.update.bind(controller)));
-router.delete("/:id", auth, reqHandler(controller.delete.bind(controller)));
+router.patch("/", auth, reqHandler(controller.update.bind(controller)));
```

Comment:
Why have you removed the id from the path?

Reply:
In this application, each email corresponds to a user, with authentication middleware ensuring user identification. I'm considering removing user IDs from routes as I perceive them as unnecessary. However, I'm open to feedback if my assumption is incorrect.